src/components/tx/TxData.js

```diff
@@ -119,6 +125,11 @@ class TxData extends React.Component {
     });
   }
 
+  getNativeToken = () => {
```

Comment:
question(non-blocking): Should we use `wallet.storage.getNativeTokenData()` instead? Or we don't have any wallets available in store?